apis/config/v1beta1/defaults.go

```diff
@@ -38,9 +36,10 @@ const (
 	DefaultLeaderElectionID                             = "c1f6bfd2.kueue.x-k8s.io"
 	DefaultClientConnectionQPS                  float32 = 20.0
 	DefaultClientConnectionBurst                int32   = 30
-	defaultPodsReadyTimeout                             = 5 * time.Minute
+	DefaultPodsReadyTimeout                             = 5 * time.Minute
 	DefaultQueueVisibilityUpdateIntervalSeconds int32   = 5
 	DefaultClusterQueuesMaxCount                int32   = 10
+	DefaultJobFrameworkName                             = "batch/job"
```

Comment:
This change might be ok, but seems unrelated, so I would suggest reverting it so that the PR is focused.

Reply:
The defaults_test.go was changed to be external to the package - `package ... _test` to avoid an import cycle. I think that is why I changed this to be exported.

Reply:
sgtm, if required to avoid the cycle